src/components/CreatePlaceFlow/pages/PointGeometryPicker.jsx

```diff
@@ -0,0 +1,164 @@
+// @flow
+import * as React from 'react';
+import styled from 'styled-components';
+import { t } from 'ttag';
+import ReactMapGL, { GeolocateControl } from 'react-map-gl';
```

Comment:
+0: What do you think about loading this with `dynamic()` as we don't have Next.js page based code splitting yet?

Reply:
I'll leave that as an improvement on top, have to restructure the components a bit.